src/App.vue

```diff
@@ -1,17 +1,194 @@
+        this.totalWrong = 0
+      }
+    },
+    assignListeners(e) {
+      if ((e.key === 'Enter') && this.isGameStarted) this.checkAnswer()
+    },
+    loadCard(nextCard) {
+      this.RhymusGame.currentCard = nextCard
+      this.guessValue = ''
+      this.hintText = ''
+      this.wrongText = ''
+      this.incorrectGuesses = 0
+      this.resetTimer()
+    },
+    resetTimer() {
+      clearInterval(this.timer)
+      this.timer = setInterval(this.countDown, 1000)
+    },
+    countDown() {
+      this.timeRemaining === 0 ? this.gameOver() : this.timeRemaining--
+    },
+    shufflePuzzles(arr1) {
+      let ctr = arr1.length
+      let index
+      let array
+      while(ctr > 0) {
+          index = Math.floor(Math.random() * ctr)
+          ctr --,
+          array = [arr1[index], arr1[ctr]] = [arr1[ctr], arr1[index]]
+      }
+      return array
+    },
+    checkAnswer() {
+      if (this.guessValue === this.RhymusGame.currentCard.answer) {
+          this.isCorrect = true
+          this.totalCorrect++
+          this.correctAnswer()
+          } else if (this.guessValue === '' || undefined) {
+          alert('Please enter a guess!')
+          } else {
+          this.isCorrect = false
+          this.totalWrong++
+          this.incorrectAnswer()
+      }
+    },
+    guessValueUpdate(val) {
+      this.guessValue = val
+    },
+    correctAnswer() {
+      this.classNameObject.cardBlock.correct = this.isCorrect ? true : null
+      this.classNameObject.guessElement.correct = this.isCorrect ? true : null
+      this.cardNumber++
+      this.correctGuesses++
+      setTimeout(() => {
+          this.classNameObject.cardBlock.correct = false
+          this.classNameObject.guessElement.correct = false
+          if (this.cardNumber < this.puzzlesArray.length) {
+              this.loadCard(this.puzzlesArray[this.cardNumber])
+          }
+          else {
+              this.gameOver()
+          }
+      }, 800)
+    },
+    incorrectAnswer() {
+      this.classNameObject.cardBlock.incorrect = this.isCorrect ? null : true
+      this.classNameObject.guessElement.incorrect = this.isCorrect ? null : true
+      this.incorrectGuesses++
+      this.guessValue = ''
+
+      setTimeout(() => {
+          this.classNameObject.cardBlock.incorrect = false
+          this.classNameObject.guessElement.incorrect = false
+      }, 500)
+      this.wrongText = `incorrect guesses: ${this.incorrectGuesses}`
+      //if there are more than 4 incorrect guesses on a single card the player loses
+      if (this.incorrectGuesses > 1 && this.incorrectGuesses < 4) {
+          this.stylingObject.hint.display = 'flex'
+          this.hintText = 'Hint: ' + this.capitalizeFirstLetter
+      } else {
+          this.gameOver()
+      }
+    },
+    gameOver() {
+        // Accounts for array starting at 0
+      if (this.timeRemaining === 0 || this.correctGuesses + 1 === this.puzzlesArray.length || this.incorrectGuesses >= 4) {
```

Comment:
Nitpick: `this.incorrectGuesses >= 4` gives the impression values can reach or exceed this condition. But of course the game stops at 4, right? ` > 3` gives a clearer meaning when reading since it comes across as a hard limit.